lib/phoenix/endpoint.ex

```diff
@@ -77,6 +77,10 @@ defmodule Phoenix.Endpoint do
 
       The default format is used when none is set in the connection.
 
+    * `:instrumentation` - a list of instrumenters modules whose callbacks will
```

Comment:
Shouldn't we call it `instrumenters`? Given the description, that seems to be what makes most sense.